lib/filter/enemy.js

```diff
@@ -55,43 +55,73 @@ exports.update = function(json) {
         rows.push(currentEnemy.lv);
         rows.push(currentEnemy.uid);
 
-        rows.push(currentEnemy.init_hp + '/' + currentEnemy.max_hp);
-
-        currentEnemy.params[0].atk = config.get('enemy.stats.atk') || currentEnemy.params[0].atk;
+        for (var param in json[round].enemy[enemy].children[part].params) {
+          currentEnemy.params[param].atk = config.get('enemy.stats.atk') || currentEnemy.params[param].atk;
+          currentEnemy.params[param].def = config.get('enemy.stats.def') || currentEnemy.params[param].def;
+          currentEnemy.params[param].matk = config.get('enemy.stats.matk') || currentEnemy.params[param].matk;
+          currentEnemy.params[param].mdef = config.get('enemy.stats.mdef') || currentEnemy.params[param].mdef;
+          currentEnemy.params[param].mnd = config.get('enemy.stats.mnd') || currentEnemy.params[param].mnd;
+          currentEnemy.params[param].acc = config.get('enemy.stats.acc') || currentEnemy.params[param].acc;
+          currentEnemy.params[param].eva = config.get('enemy.stats.eva') || currentEnemy.params[param].eva;
+          currentEnemy.params[param].spd = config.get('enemy.stats.spd') || currentEnemy.params[param].spd;
+        
+          var buffs = '';
+          if (config.get('enemy.defAttributes.enabled')) {
+            var NewDefAttrib = [];
```

Comment:
Please use lowercase for the first character `newDefAttr` or something like that.